crates/rolldown/tests/esbuild/importstar_ts/ts_re_export_type_only_file_es6/_config.json

```diff
@@ -5,7 +5,7 @@
         "name": "entry",
         "import": "entry.ts"
       }
-    ]
-  },
-  "expectError": true
+    ],
+    "shimMissingExports": true
```

Comment:
@IWANABETHATGUY cc. Rolldown doesn't support/detect `export type` natively, so `shimMissingExports` is an workaround for such case.